server/graphql/v2/object/Comment.ts

```diff
@@ -0,0 +1,40 @@
+import { GraphQLInt, GraphQLString, GraphQLObjectType } from 'graphql';
+import { strip_tags } from '../../../lib/utils';
+
+import { DateString } from '../scalar/DateString';
+import { Account } from '../interface/Account';
+
+const Comment = new GraphQLObjectType({
+  name: 'Comment',
+  description: 'This represents an Comment',
+  fields: () => {
+    return {
+      id: {
+        type: GraphQLInt,
+      },
```

Comment:
As a design principle of the API v2, we decided to not expose the internal identifiers anymore. We're generally using obfuscated ids as replacement, which is not perfect but it's like that.

Reply:
Oh I see. That is what this code is for: `return idEncode(transaction.id, 'transaction');`. Ok, I'll make the change there.

Reply:
Done. I create a new function called `idEncodeResolver`.  It receives a type and returns a resolver that does the encoding.